nxcompshad/src/Core.h

```diff
@@ -88,9 +88,9 @@ class CorePoller
 
   unsigned int bpl_;
 
-  unsigned int width_;
+  unsigned short int width_;
```

Comment:
> Doing that sounds like a bad idea, since CorePoller is used as a base class for the specialized (Win and X11) Poller classes.
> [...]

Ok, I played with this a little (as discussed on IRC). It seems that we will add the -Wconversion flag to our build options to catch the whole mess behind this. This will happen in a separate PR.

For now, I have dropped the commit in question from this PR and moved it over to a new PR-in-prep.